docs/migrating/from-gatsby.md

```diff
@@ -0,0 +1,235 @@
+---
+description: Learn how to transition an existing Gatsby project to Next.js.
+---
+
+# Migrating from Gatsby
+
+This guide will help you understand how to transition from an existing Gatsby project to Next.js. Since both frameworks are built on top of React, the transition can be broken down into a series of steps. Migrating to Next.js will allow you to:
+
+- Choose which [data fetching](/docs/basic-features/data-fetching) strategy you want on a per-page basis (instead of only SSG)
```

Comment:
Let's not use `SSG` here. Right now we don't mention SSG anywhere in our docs and this shouldn't be the first place we use it.